content/en/tracing/guide/service_overrides.md

```diff
@@ -133,6 +133,14 @@ Similarly, for a span representing a call to a mySQL database:
 
 **Note**: The configuration above only removes [integration service overrides](#integration-service-overrides). Custom service overrides must be removed directly in the code.
 
+## Glossary
```

Comment:
Having a small glossary for this page alone limits its use. Can you add these terms to the main glossary instead? You can see it here: https://docs.datadoghq.com/glossary/.

Here are instructions for adding a term to the glossary: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/3154742084/Glossary#Creating-a-glossary-entry. If you don't have the documentation build set up locally, ask for help in Slack in #documentation.

Reply:
I'd like to keep this guide isolated ideally, as these terms are not used anywhere else for now. Once we move this page to the main docs, I'll move these terms to the global glossary